test/wrb.js

```diff
@@ -360,13 +358,13 @@ contract("WitnetRequestBoard", ([
           from: other,
           gasPrice: 1,
         }),
-        "The dr has been manipulated and the bytecode has changed"
+        "manipulated request"
```

Comment:
This was confusing to me: "manipulated" means that the data request bytecode was manipulated, not that the result was manipulated. I would prefer a message like "data request bytecode has changed", because "manipulated" gives the impression that witnet can be manipulated easily and it's not secure.

Reply:
Ok. This "manipulated request" text is part of revert messages produced at the "WitnetRequestBoard" contract. It will be changed to "bytecode changed after posting"